### PR TITLE
Ruthvik presentation

### PR DESCRIPTION
**Pull request by yashb196**
Hello Everyone, I have added the report for the presentation by @ruthvikm  on April 5th at the pius library please do have a look at it and do let me know if there is any feedback needed on that regard

### CODE REVIEW

File: blog/2024-04-05-pi4micronaut.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+image: ./pi4micronaut-20240314/Baking_Pi4Micronaut.png
+hide_table_of_contents: false
+---
+
+Ruthvik presented the Pi4Micronaut. Students got a hands-on practice using the pi4micronaut library. It was a great success that exceeded the expectations of all the planners. 
+
+<!--truncate-->
+
+**What:** Pi4 Micronaut <br/>
+**Who:** Presentedy by [Ruthvik Mannem](https://github.com/ruthvikm), MS student in Computer Science<br/>
```
Comment (ruthvikm) [inline blog/2024-04-05-pi4micronaut.md:16]: MS student in Artificial Intelligence

File: blog/2024-04-05-pi4micronaut.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+**Resources:** [presentation of instructions](https://oss-slu.github.io/Pi4Micronaut/)
+
+A group of students and staff huddled over laptops around a large conference table in the Academic Technology Commons of Saint Louis University's Pius Library.  On an already busy Friday afternoon these students had elected to spend their time getting some hands on experiences with open source software. 
+
+Ruthvik Mannem, a second year Masters student in Computer Science and a Research Assistant with Open Source with SLU, Explained about how he had created a library for micronaut which worked on Java instead of Python. Then he went [step-by-step](https://docs.google.com/presentation/d/1etn4jDqLUA8tdQhA8emUGaOXri8pRJKu/edit?usp=sharing&ouid=107304909661279551530&rtpof=true&sd=true) with students down the path of installing the libraries and packages to communicate with the micronaut  ![Raspberry Pi 3](./oss-atc-20240405/20240405_3.jpeg). The Library does support Ruthvik then walked those attending the workshop through actually using the LED Function along with the Bulb Funcationality they created as a [Example](./oss-atc-20240405/20240405_4.jpeg).
```
Comment (ruthvikm) [inline blog/2024-04-05-pi4micronaut.md:24]: Masters student in Artificial Intelligence

**Review by ruthvikm — CHANGES_REQUESTED** [review]
Requesting to change few typos. Also "OpenSourcewithSLU" is a single word as per Daniel's previous review.

**Review by ruthvikm — DISMISSED** [review]

**Comment by kungfuchicken** [comment]
> Requesting to change few typos. Also "OpenSourcewithSLU" is a single word as per Daniel's previous review.

should be "Open Source with SLU"

File: blog/2024-04-05-pi4micronaut.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+**Resources:** [presentation of instructions](https://oss-slu.github.io/Pi4Micronaut/)
+
+A group of students and staff huddled over laptops around a large conference table in the Academic Technology Commons of Saint Louis University's Pius Library. On an already busy Friday afternoon these students had elected to spend their time getting some hands on experiences with open source software. 
+
+Ruthvik Mannem, a second year Masters student in Artificial Intelligence and a Research Assistant with OpenSourcewithSLU, Explained about how he had created a library for micronaut which worked on Java instead of Python. Then he went [step-by-step](https://docs.google.com/presentation/d/1etn4jDqLUA8tdQhA8emUGaOXri8pRJKu/edit?usp=sharing&ouid=107304909661279551530&rtpof=true&sd=true) with students down the path of installing the libraries and packages to communicate with the micronaut  ![Raspberry Pi 4](./oss-atc-20240405/20240405_3.jpeg). The Library works upto Raspberry Pi 4B  Ruthvik then walked those attending the workshop through actually using the LED/LCD Functions along with the Bulb Functionality they created as a [Example](./oss-atc-20240405/20240405_4.jpeg).
```
Comment (kungfuchicken) [inline blog/2024-04-05-pi4micronaut.md]: Micronaut should be capitalized. I think this maybe should read "to communicate with the Micronaut library." (end with "library" and put the period before the inline image.) 

"up to"

"as an [example]"

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
looks great! there are a few small edits i'd like.

File: blog/2024-04-05-pi4micronaut.md (hunk 3 above)
Comment (kungfuchicken) [reply blog/2024-04-05-pi4micronaut.md]: maybe "Micronaut framework" instead :D

**Review by kungfuchicken — APPROVED** [review]
lgtm; 🚀

**Pull request merged**